script/authorization.js

```diff
@@ -36,16 +36,19 @@ let render = () => {
   titlePassword.textContent = "Password";
   titlePassword.id = "titleField";
 
-  const buttomLogin = document.createElement("button");
-  buttomLogin.textContent = "Войти";
-  buttomLogin.id = "buttomLogin";
+  const buttonLogin = document.createElement("button");
+  buttonLogin.textContent = "Войти";
+  buttonLogin.id = "buttonLogin";
+  buttonLogin.onclick = () => {
+    window.location.href = "personalArea.html";
+  };
```

Comment:
I don't see where you send the request to the server? and where is checked for login and password?